eng/ci/templates/official/jobs/build-artifacts-windows.yml

```diff
@@ -51,7 +46,7 @@ jobs:
     inputs:
       command: custom
       custom: pack
-      arguments: -p:BuildNumber=$(buildNumber) -c release $(packSuffixSwitch) -o $(nuget_package_path)
```

Comment:
Don't we need to pass `BuildNumber` argument anymore?

Reply:
Nope! The versioning refactor has this all calculated in msbuild.

1. `BuildNumber` is now a "short date" calculation (taken from dotnet's versioning scheme). `SHORT_DATE := yy * 1000 + MM * 50 + dd `. The need for this calculation is to ensure it is less than 65,535, which is a windows limitation for build numbers.
2. I also have incorporation for a "build counter" to differentiate similar short dates, but that may need more testing/work (not super important). It is supposed to be an incrementing counter from 0 for the same day. so `{ShortDate}.{Counter}`